src/sentry/static/sentry/app/components/charts/utils.jsx

```diff
@@ -56,6 +56,28 @@ export function getInterval(datetimeObj, highFidelity = false) {
   }
 }
 
+/**
+ * Convert an interval string into a number of seconds.
+ * This allows us to create end timestamps from starting ones
+ * enabling us to find events in narrow windows.
+ *
+ * @param {String} interval The interval to convert.
+ * @return {Integer}
+ */
+export function intervalToMilliseconds(interval) {
```

Comment:
We have this to hours: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils.jsx#L246

Reply:
I can move this to live with its friend. I wonder if both shouldn't live in utils/dates.jsx?

Reply:
I personally find the util functions hard to discover when they are grouped together in the same files, but moving them into `utils/dates` is an improvement 👍